obspy/io/gcf/__init__.py

```diff
@@ -18,11 +19,83 @@
 The format will be determined automatically.
 
 >>> from obspy import read
->>> st = read("/path/to/20160603_1955n.gcf")
+>>> st = read("/path/to/20160603_1955n.gcf")  #doctest: +ELLIPSIS
```

Comment:
This "#doctest: +ELLIPSIS" doesn't do anything. "Ellipsis" modifier is for allowing "..." as placeholder when testing printed text output, but this command does not have any text output.

Reply:
Ok, fixed (pretty much at copy-paste-hammer level when it comes to the sphinx syntax, sorry)